genesis-programs/src/non_circulating_accounts.rs

```diff
@@ -0,0 +1,30 @@
+pub(crate) const NON_CIRCULATING_ACCOUNTS: [&str; 25] = [
```

Comment:
Some brief comments in this file that mark these accounts as mainnet-beta accounts would be nice.